packages/firebase_core/firebase_core_web/lib/src/firebase_core_web.dart

```diff
@@ -75,118 +75,62 @@ class FirebaseCoreWeb extends FirebasePlatform {
 
   /// Injects a `script` with a `src` dynamically into the head of the current
   /// document.
-  Future<void> _injectSrcScript(String src) async {
+  Future<void> _injectSrcScript(String src, String windowVar) async {
     ScriptElement script = ScriptElement();
     script.type = 'text/javascript';
```

Comment:
Can we add `script.crossorigin = 'anonymous';` here as well. Without this currently unable to capture errors that originate from inside the JS, e.g. if I have some Browser/JS error reporting tool integrated into my website all errors that originate from inside a Firebase SDK will appear as  just `"Script error."` and no extra information

More context; https://blog.sentry.io/2016/05/17/what-is-script-error

Ping me internally if a repro is needed (I have one on an internal tool)